runtime-sdk/src/modules/accounts/mod.rs

```diff
@@ -422,6 +426,41 @@ impl module::Module for Module {
 }
 
 impl module::MethodHandler for Module {
+    fn prefetch(
+        prefixes: &mut BTreeSet<Prefix>,
+        method: &str,
+        body: cbor::Value,
+        auth_info: &AuthInfo,
+    ) -> module::DispatchResult<cbor::Value, Result<(), error::RuntimeError>> {
+        match method {
+            "accounts.Transfer" => {
+                module::DispatchResult::Handled(|| -> Result<(), error::RuntimeError> {
+                    let args: types::Transfer =
```

Comment:
One annoying thing seems to be that we will now be deserializing the body twice, once during `prefetch` and again in `dispatch_call`. Not sure how hard it would be to avoid double deserialize, maybe as a future improvement :-)

Reply:
Hm yeah, i'll open an issue for this once merged.